modules/hcp-disconnected-mgmt-cluster.adoc

```diff
@@ -0,0 +1,141 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-deploy-disconnected.adoc
+
```

Comment:
Question for technical reviewers: Do these steps apply only to virtual environments?